pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -51,7 +51,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	"sigs.k8s.io/controller-runtime/pkg/source"
```

Comment:
did we have to drop the package here and add the prefix below ?

Reply:
we drooped it because it's not needed. 
the optimization done automatically for deps which are not in used.